helm/README.md

```diff
@@ -0,0 +1,217 @@
+# Prometheus
+
+An opinionated [Prometheus](https://prometheus.io/) Helm chart for deployment in Giant Swarm clusters.
+
+## Prerequisites
+
+- Kubernetes 1.3+ with Beta APIs enabled
+- [Helm](https://helm.sh) package manager.
+
+## Installing the Chart
+
+To install the chart:
+
+```console
+$ helm install ./helm --name prometheus
```

Comment:
should we change `prometheus` to `my-prometheus` to show that you can use your own naming here?

Reply:
good idea